docs/components/tools.md

```diff
@@ -1,18 +1,75 @@
 ---
-title: Bucket
+title: Tools
 type: docs
 menu: components
 ---
 
-# Bucket
+# Tools
 
-The bucket component of Thanos is a set of commands to inspect data in object storage buckets.
-It is normally run as a stand alone command to aid with troubleshooting.
+The `thanos tools` subcommand of Thanos is a set of additional CLI, short-living tools that
+are meant to be ran for development or debugging purposes.
+
+All commands added as tools should land in `tools.go` or file with `tools_` prefix.
+
+## Flags
+
+[embedmd]:# (flags/tools.txt $)
+```$
+usage: thanos tools <command> [<args> ...]
+
+Tools utility commands
+
+Flags:
+  -h, --help               Show context-sensitive help (also try --help-long and
+                           --help-man).
+      --version            Show application version.
+      --log.level=info     Log filtering level.
+      --log.format=logfmt  Log format to use. Possible options: logfmt or json.
+      --tracing.config-file=<file-path>
+                           Path to YAML file with tracing configuration. See
+                           format details:
+                           https://thanos.io/tracing.md/#configuration
+      --tracing.config=<content>
+                           Alternative to 'tracing.config-file' flag (lower
+                           priority). Content of YAML file with tracing
+                           configuration. See format details:
+                           https://thanos.io/tracing.md/#configuration
+
+Subcommands:
+  tools bucket verify [<flags>]
+    Verify all blocks in the bucket against specified issues
+
+  tools bucket ls [<flags>]
+    List all blocks in the bucket
+
+  tools bucket inspect [<flags>]
+    Inspect all blocks in the bucket in detailed, table-like way
+
+  tools bucket web [<flags>]
+    Web interface for remote storage bucket
+
+  tools bucket replicate [<flags>]
+    Replicate data from one object storage to another. NOTE: Currently it works
+    only with Thanos blocks (meta.json has to have Thanos metadata).
+
+  tools bucket downsample [<flags>]
+    continuously downsamples blocks in an object store bucket
+
+  tools rules-check <rule-files>...
+    Check if the rule files are valid or not.
+
+
+```
+
+## Bucket
+
+The `thanos tools bucket` subcommand of Thanos is a set of commands to inspect data in object storage buckets.
+It is normally run as a standalone command to aid with troubleshooting.
 
 Example:
 
 ```bash
-$ thanos bucket verify --objstore.config-file=bucket.yml
+thanos tols bucket verify --objstore.config-file=bucket.yml
```

Comment:
```suggestion
$ thanos tools bucket verify --objstore.config-file=bucket.yml
```

Reply:
Maybe we should keep `$` to be consistent.

Reply:
In general I think we should try not to prefix with `$` since it makes it harder to copy and paste example commands

Reply:
yea TBH I am not sure how this even got there.. why is this convention?

Reply:
Removing all